llvm/lib/Support/Unix/Threading.inc

```diff
@@ -338,7 +344,8 @@ llvm::BitVector llvm::get_thread_affinity_mask() {
 
 unsigned llvm::get_cpus() { return 1; }
 
-#if defined(__linux__) && (defined(__i386__) || defined(__x86_64__))
+#if (defined(__linux__) || defined(__CYGWIN__)) &&                             \
+    (defined(__i386__) || defined(__x86_64__))
 // On Linux, the number of physical cores can be computed from /proc/cpuinfo,
```

Comment:
This tries to read `/proc/cpuinfo` - I don't expect cygwin to provide that, or does it?

Reply:
Oh, that's fascinating that they've emulated that too!